README.md

```diff
@@ -25,6 +25,8 @@
 
 ## Overview
 
+**Note:** Documents in this repo are in pre-release version. For the released version, you can refer to [IBM Knowledge Center](https://www.ibm.com/support/knowledgecenter/SSHKN6/installer/1.x.x/index.html)
```

Comment:
**Note:** Documents in this repo are a pre-release version. For the release version, see [IBM Knowledge Center](https://www.ibm.com/support/knowledgecenter/SSHKN6/installer/1.x.x/index.html).